Core/resources/template.html

```diff
@@ -0,0 +1,18 @@
+template on ma-core-public core resources
```

Comment:
This file shouldn't be in the delivered jar.  If you use it for testing put it into resources-test

Reply:
removed.